docs/INSTALLATION.md

```diff
@@ -32,4 +32,3 @@ that will give you the Visual Studio 2013 IDE and the latest version of the
 Microsoft Visual C compiler.
 
 [vs-1013]: http://www.visualstudio.com/en-us/products/visual-studio-community-vs
```

Comment:
Good deal, I'll switch it over. Honestly I have no idea how much of a nightmare this is on Windows as I avoid it whenever possible :)